tests/test_webhook.py

```diff
@@ -133,15 +139,41 @@ def test_fails_on_not_known_action(self):
 
     def test_extract_browser_label(self):
         """Extract browser label name."""
-        browser_label = helpers.extract_browser_label(self.issue_body)
+        browser_label = helpers.extract_browser_label(
+            helpers.extract_metadata(self.issue_body))
         self.assertEqual(browser_label, 'browser-firefox')
-        browser_label_none = helpers.extract_browser_label(self.issue_body2)
+        browser_label_none = helpers.extract_browser_label(
+            helpers.extract_metadata(self.issue_body2))
```

Comment:
I guess we had relied on the notion that `<!-- @browser: Foobar -->` would be invalid, because it doesn't have a version number. I don't think anything would actually break, if we allowed it. At least not for labels.

But I had to enforce this in `extract_browser_label`, and added the extra `issue_body5` to test this (as well as test the ordering stuff).

Reply:
see my comments in creating the metadata_dict

Reply:
> see my comments in creating the metadata_dict

Yeah, saw that. I'm going to leave it as is for now. Thanks.